include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -6026,8 +6026,9 @@ struct OrtApi {
    */
   ORT_API2_STATUS(Node_GetGraph, _In_ const OrtNode* node, _Outptr_result_maybenull_ const OrtGraph** graph);
 
-  /** \brief Returns the execution provider type (name) that this node is assigned to run on.
+  /** \brief Returns the execution provider name that this node is assigned to run on.
    *         Returns NULL if the node has not been assigned to any execution provider yet.
+   *         For plugin execution providers, the name is the one registered via RegisterExecutionProviderLibrary.
```

Comment:
RegisterExecutionProviderLibrary uses a registration name but that isn't necessarily the EP name as a library can provide multiple EP factories. It is in the example EP though as they're 1:1.

The EP name primarily comes from OrtEpFactory::GetName (the OrtEpFactory populates the name in the OrtEpDevice), but it's also available from OrtEp::GetName. 

As this code is relevant to the EP instance, I think the most accurate to say here is that it's the value returned by OrtEp::GetName.

```suggestion
   *         For plugin execution providers, the name is the one returned by OrtEp::GetName.
```